tests/per_usage/divs_html_elements/scripts/awesome-doc-styling.js

```diff
@@ -0,0 +1,26 @@
+let is_stylished = false;
```

Comment:
I'd rather name that file "style-mutators" rather than "styling", so the purpose is explicit